src/network/networkd-network-gperf.gperf

```diff
@@ -280,6 +280,7 @@ IPv6AcceptRA.UseOnLinkPrefix,                config_parse_bool,
 IPv6AcceptRA.UseDNS,                         config_parse_bool,                                        0,                             offsetof(Network, ipv6_accept_ra_use_dns)
 IPv6AcceptRA.UseDomains,                     config_parse_ipv6_accept_ra_use_domains,                  0,                             offsetof(Network, ipv6_accept_ra_use_domains)
 IPv6AcceptRA.UseMTU,                         config_parse_bool,                                        0,                             offsetof(Network, ipv6_accept_ra_use_mtu)
+IPv6AcceptRA.UseHopLimit,                    config_parse_bool,                                        0,                             offsetof(Network, ipv6_accept_ra_use_hop_limit)
```

Comment:
Avoid a configuration option when an RFC specifies the feature as a "SHOULD" requirement.